packages/@okta/vuepress-site/docs/guides/style-the-widget/style-okta-hosted/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Style the Okta-hosted Sign-In Widget
+---
+
+You can add any HTML, CSS, or JavaScript to the **Customization** page that you want. You can also customize the sign-in page per application and with multiple brands.
+
+> **Note:** Before you can get started customizing the Okta-hosted sign-in page, you must have already customized your [Okta URL domain](/docs/guides/custom-url-domain/).
+
+## Edit the sign-in page
+
+The **Customize Sign-In** page offers both basic and advanced customization options to create a completely transformed sign-in experience. To access this page, from the Developer Console select **Customization**, and then **Signin Page**.
```

Comment:
The title on this page is "Customize Sign-in Page", so maybe Page should be included in bold?